src/bootstrap/doc.rs

```diff
@@ -375,6 +376,7 @@ impl Step for Standalone {
                up_to_date(&footer, &html) &&
                up_to_date(&favicon, &html) &&
                up_to_date(&full_toc, &html) &&
+               version_info.exists() &&
```

Comment:
This can be false during the dry run. I think the correct fix would be to not run `up_to_date(&version_info, &html)` if `builder.config.dry_run` just like `up_to_date(&rustdoc, &html)` below.